platforms/core-configuration/kotlin-dsl/build.gradle.kts

```diff
@@ -123,7 +133,7 @@ dependencies {
     testFixturesImplementation(libs.jacksonKotlin)
     testFixturesImplementation(libs.asm)
 
-    integTestDistributionRuntimeOnly(project(":distributions-basics"))
+    integTestDistributionRuntimeOnly(project(":distributions-full"))
```

Comment:
❓ Why change from `basics` to `full` here?

Reply:
All of this was just trying things to see if any of them fixed that error.